cfpb/consumerfinance.gov
# Paying for College: Remove unused methods, variables, and student-debt-calc module

## Removals

- Removes imported paying for college app methods that aren't referenced.
- Removes unused `body` parameter from financial-view `init` method.
- Reorders imports to satisfy linter.
- Removes unused variables `isRate` and `isFee` in expenses-view.
- Removes unused variables `offerContent` and `costsContent` in financial-view.
- Removes unused `student-debt-calc` module.

## How to test this PR

1. Pull, build, and run http://localhost:8000/paying-for-college/your-financial-path-to-graduation/ and it should be unchanged from production.
